### PR TITLE
Add Scorum network

### PR DESCRIPTION
**Pull request by mikluke**

### CODE REVIEW

File: scorum/chain.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  "daemon_name": "scorumd",
+  "node_home": "$HOME/.scorum",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "gas",
```
Comment (JeremyParish69) [inline scorum/chain.json:18]: This denom must be defined as a base denom in the assetlist

File: scorum/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      "name": "Scorum Power",
+      "display": "sp",
+      "symbol": "SP",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
```
Comment (JeremyParish69) [inline scorum/assetlist.json:54]: These images do not exist. P{lease upload scorum.png and scorum.svg (unless you mean to point to 'sp.png' and 'sp.svg', in which can please correct this and the image reference in chain.json

File: scorum/chain.json (hunk 1 above)
Comment (mikluke) [reply scorum/chain.json:18]: fixed

File: scorum/assetlist.json (hunk 2 above)
Comment (mikluke) [reply scorum/assetlist.json]: fixed

File: scorum/chain.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        "name": "v1.0.0",
+        "recommended_version": "v1.0.0",
+        "compatible_versions": [
+          "v1.0.0"
+        ]
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/scorum/images/scorum.png",
```
Comment (JeremyParish69) [inline scorum/chain.json:53]: This images do not exist. Please update the reference (same with the `images` property

Comment (mikluke) [reply scorum/chain.json:53]: Uploaded logo for the chain

**Pull request merged**